contrib/question_converters/canvas/quiz2pl.py

```diff
@@ -93,15 +93,32 @@ def file_name_only(name):
 if quiz["time_limit"]:
     pl_quiz["allowAccess"][0]["timeLimitMin"] = quiz["time_limit"]
 
+
+def clean_question_text(text: str) -> str:
+    # Some Canvas plugins like DesignPlus inject custom CSS and JavaScript into
+    # all questions. This code is not needed in PrairieLearn, and in fact can
+    # cause problems in CBTF environments since they'll be forbidden from loading
+    # by most firewalls/proxies. We just remove them.
+    #
+    # We use regex instead of a proper HTML parser because we want to limit this
+    # script to only using the Python standard library.
+    text = re.sub(r"<link[^>]*>", "", text)
+    text = re.sub(r"<script[^>]*>.*?</script>", "", text)
```

Comment:
Shouldn't we just ban external scripts?

Reply:
Why _would_ we want to keep inline scripts?

Reply:
I have no idea, maybe there is a plugin that injects this stuff as a inline script, in which case it might still work?

Reply:
Sorry, this is a nit, not blocking.

Reply:
Yeah, I think there's basically no hope of Canvas JS plugins doing anything useful in PL, so I think a blanket removal of all JS is fine.